src/components/block/FooterCTACard/FooterCTACard.module.css

```diff
@@ -0,0 +1,28 @@
+.body {
+  color: var(--color-white);
+  font: var(--font-body-medium);
+  line-height: 1.3;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
+  max-width: 350px;
+  text-align: center;
+}
+
+.button {
```

Comment:
This class name a little bit misleading because it's not actually used on the button itself, but rather the block-level element that wraps it. How about you rename this to `.buttonWrapper`?